compat/dev_to_tty.c

```diff
@@ -30,7 +30,7 @@
 #include <fcntl.h>
 #include <unistd.h>
 
```

Comment:
MSVC has problems with this file (it doesn't know `unistd.h`, doesn't support `restrict` keyword, dosn't have `readlink`, etc.). But it is used only in Unix probes, so it should be enough to remove it  from Windows builds.

Reply:
fixed